data/json/items/comestibles/carnivore.json

```diff
@@ -106,7 +107,7 @@
     "parasites": 0,
     "healthy": 0,
     "fun": 0,
-    "flags": [ "EATEN_HOT" ]
+    "flags": [ "EATEN_HOT", "TRADER_AVOID" ]
```

Comment:
I removed the flag from items in that file. Its indeed inconsistently used there.